--- trilinos-11.4.1-Source/packages/kokkos/classic/NodeAPI/CMakeLists.txt	2013-07-09 21:18:24.000000000 +0000
+++ trilinos-11.4.1-Source.altered/packages/kokkos/classic/NodeAPI/CMakeLists.txt	2013-12-05 12:54:51.261852236 +0000
@@ -25,6 +25,7 @@
     Kokkos_CUDANodeMemoryModelImpl.hpp
     Kokkos_ThrustGPUNode.cuh
     Kokkos_ThrustGPUNode.hpp
+    Kokkos_ThrustGPUWrappers.hpp
     Kokkos_CUDANodeUtils.hpp
 )
 APPEND_SET(CUDA_SOURCES
