--- trilinos-11.4.1-Source/cmake/TPLs/FindTPLCUSPARSE.cmake	2013-07-09 21:17:13.000000000 +0000
+++ trilinos-11.4.1-Source.altered/cmake/TPLs/FindTPLCUSPARSE.cmake	2013-12-04 15:32:57.097848788 +0000
@@ -55,6 +55,7 @@
 
 # Check for CUDA support
 INCLUDE(TribitsTplDeclareLibraries)
+MESSAGE(${CUDA_VERSION})
 
 IF (NOT TPL_ENABLE_CUDA OR CUDA_VERSION VERSION_LESS "4.1")
   MESSAGE(FATAL_ERROR "\nCUSPARSE: did not find acceptable version of CUDA libraries (4.1 or greater)")
@@ -63,7 +64,7 @@
     # FindCUDA before CMake 2.8.8 does not find cusparse library; therefore, we must
     find_library(CUDA_cusparse_LIBRARY
       cusparse
-      HINTS ${CUDA_TOOLKIT_ROOT_DIR}/lib
+      HINTS ${CUDA_TOOLKIT_ROOT_DIR}/lib64
       )
     IF(CUDA_cusparse_LIBRARY STREQUAL "CUDA_cusparse_LIBRARY-NOTFOUND") 
       MESSAGE(FATAL_ERROR "\nCUSPARSE: could not find cuspasre library.")
