--- trilinos-11.4.1-Source/packages/kokkos/classic/LinAlg/Kokkos_CUSPARSEOps.hpp	2013-07-09 21:18:24.000000000 +0000
--- trilinos-11.4.1-Source.altered/packages/kokkos/classic/LinAlg/Kokkos_CUSPARSEOps.hpp	2014-02-04 15:45:26.798468570 +0000
+++ b/Kokkos_CUSPARSEOps.hpp	2014-02-04 15:48:37.390465632 +0000
@@ -1054,7 +1054,7 @@
     RCP<const cusparseHandle_t> hndl = CUSPARSEdetails::Session::getHandle();
     // look at the parameter list and do any analyses requested for solves
     RCP<cusparseSolveAnalysisInfo_t> ai_non, ai_trans, ai_conj;
-    if (params != null && params->get("Prepare Solve",false)) {
+    if (params != null && params->get("Prepare Solve",true)) {
       ai_non = CUSPARSEdetails::createSolveAnalysisInfo();
       cusparseStatus_t stat =
           CUSPARSEdetails::CUSPARSETemplateAdaptors<Scalar>::CSRSM_analysis(
