--- trilinos-11.4.1-Source/packages/ifpack2/test/belos/hybrid_platform/BelosTpetraHybridPlatformExample.cpp	2013-07-09 21:18:12.000000000 +0000
+++ trilinos-11.4.1-Source.altered/packages/ifpack2/test/belos/hybrid_platform/BelosTpetraHybridPlatformExample.cpp	2013-12-05 09:19:11.089848625 +0000
@@ -131,7 +131,7 @@
   
     ThreadedBlasKiller<Node>::kill();
 
-    typedef double Scalar;
+    typedef float Scalar;
     typedef int LO; //LocalOrdinal
     typedef int GO; //GlobalOrdinal
     typedef Tpetra::MultiVector<Scalar,LO,GO,Node> TMV;
